app/views/support_interface/application_forms/show.html.erb

```diff
@@ -4,6 +4,20 @@
 
 <%= render SupportInterface::ApplicationSummaryComponent.new(application_form: @application_form) %>
 
+<div class="govuk-button-group">
+  <% if (recommended_courses_url = CandidateInterface::CoursesRecommender.recommended_courses_url(candidate: @application_form.candidate, locatable: @application_form).presence) %>
```

Comment:
I don't think you'll ever get to the else statement. The rest of the query string will still be built, even if it's not locatable, right?

Reply:
Updated the logic to potentially return a nil for the URL - I flip-flopped on this all the way throughj